web/Services/TenantService.cs

```diff
@@ -18,6 +20,14 @@ public TenantService(BricksAndHeartsDbContext dbContext, IPostcodeService postco
         _dbContext = dbContext;
         _postcodeService = postcodeService;
     }
+    
+    public TenantCountModel CountTenants()
+    {
+        TenantCountModel tenantCounts = new TenantCountModel();
+        tenantCounts.RegisteredTenants = _dbContext.Tenants.Count();
+        tenantCounts.LocatedTenants = _dbContext.Tenants.Count(t => t.Postcode != null);
+        return tenantCounts;

```

Comment:
Equally this can be replaced with 
```suggestion
        return new TenantCountModel
        {
            RegisteredTenants = _dbContext.Tenants.Count(),
            LocatedTenants = _dbContext.Tenants.Count(t => t.Postcode != null)
        };

```